src/alpine/3.12/helix/amd64/Dockerfile

```diff
@@ -24,12 +24,14 @@ RUN apk update && \
         libffi \
         libffi-dev \
         linux-headers \
+        lldb \
```

Comment:
This is redundant since `lldb-dev` is already listed and `lldb` is a dependency of `lldb-dev`.  Same applies to the Alpine 3.13 Dockerfile.

Reply:
I know it was redundant but I wanted to be clear that lldb itself is needed to run tests instead of just for building.

Reply:
Is leaving it ok with you?